Index: i3status/src/print_wireless_info.c
===================================================================
--- i3status.orig/src/print_wireless_info.c
+++ i3status/src/print_wireless_info.c
@@ -64,7 +64,9 @@
 
 typedef struct {
     int flags;
+#ifdef IW_ESSID_MAX_SIZE
     char essid[IW_ESSID_MAX_SIZE + 1];
+#endif
 #ifdef LINUX
     uint8_t bssid[ETH_ALEN];
 #endif
@@ -518,9 +520,11 @@ void print_wireless_info(yajl_gen json_g
         }
 
         if (BEGINS_WITH(walk + 1, "essid")) {
+#ifdef IW_ESSID_MAX_SIZE
             if (info.flags & WIRELESS_INFO_FLAG_HAS_ESSID)
                 maybe_escape_markup(info.essid, &outwalk);
             else
+#endif
                 *(outwalk++) = '?';
             walk += strlen("essid");
         }
